src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.BuildOrder.targets

```diff
@@ -22,6 +22,7 @@ projects, these properties are set in Xamarin.Android.Legacy.targets.
       _CleanIntermediateIfNeeded;
       _CheckProjectItems;
       _CheckForContent;
+      _CheckForObsoleteFrameworkAssemblies;
```

Comment:
This is causing tests to fail under One.NET, e.g. https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=4679171&view=ms.vss-test-web.build-test-results-tab&runId=20951028&resultId=100036&paneView=attachments

```
/Users/runner/Library/Android/dotnet/sdk/6.0.100-preview.4.21215.1/Microsoft.Common.CurrentVersion.targets(882,7):
error MSB4057: The target "_CheckForObsoleteFrameworkAssemblies" does not exist in the project. 
[/Users/runner/work/1/s/bin/TestRelease/temp/DuplicateRJavaOutput/UnnamedProject.csproj]
```

`_CheckForObsoleteFrameworkAssemblies` was only defined in `Xamarin.Android.Legacy.targets`: https://github.com/xamarin/xamarin-android/pull/5755/files#diff-47d20207182aad2c752c0fa397c77bd67457c3cda762b716be47e8d45c1ccda5R230

Perhaps `Microsoft.Android.Sdk.BuildOrder.targets` shouldn't be modified at all?